docs/azmcp-commands.md

```diff
@@ -505,8 +505,8 @@ azmcp sql db show --subscription <subscription> \
 
 ### Azure SQL Server Operations
 ```bash
-# List Active Directory administrators for a SQL server
-azmcp sql server adadmin list --subscription <subscription> \
+# List Microsoft Entra ID administrators for a SQL server
+azmcp sql server entraadmin list --subscription <subscription> \
```

Comment:
Is this preferred over `entraidadmin`? I don't hold a strong opinion either way, I just noticed we replaced AD with Entra ID in a bunch of docs but replaced `ad` with `entra` in the command and related classes.

Reply:
@AV25242 ?

Reply:
@ericshape here is my thought , below is how its displayed in Azure SQL UI, ideally I would have preferred entra-admin because its not any id but an admin, however with the limitation of not being able to have a hyphen I am ok with entraadmin as its truly administrator at this point

![image](https://github.com/user-attachments/assets/97287278-b0e4-44e0-bb16-6d27e19669cb)